van-gogh-second-page/maria/index-Van-Gogh-for-Children.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Van Gogh Gallery</title>
+    <link rel="stylesheet" href="index-Van-Gogh-for-Children.css" />
+   </head>
+
+<body>
+    <div class="everything-container">
+    <header>
+        <nav>
+            <a class="nav-tab">
+                Home
+            </a>
+            <p class="nav-tab">
+                Tickets
+            </p>
+            <p class="nav-tab">
+                Art and stories
+            </p>
+            <p class="nav-tab">
+                About
+            </p>
+            <p class="nav-tab">
+                Webstore
+            </p>
+            <p class="nav-tab">
+                NL|EN
+            </p>
+            <p class="nav-tab">
+                Search
+            </p>
+        </nav>
+        <div id="van-gogh-logo">
+            Van
+            <br />
+            Gogh
+            Museum
+            <span style="font-weight: 200; font-size: 9px;">Amsterdam</span>
+        </div>
+    </header>
+
+    <div class="title">
+        <h1>Van Gogh for <br> Children</h1>
+        <p class="title-description">Fun Van Gogh inspired art activities for children of all ages!</p>
+    </div>
+
+    <div class="navigation">
+        <div class="navigation-item"> <a href="#container-1"> Who was Vincent? ↓ </a></div>
+        <div class="navigation-item"><a href="#container-2"> Colour a Masterpiece ↓ </a></div>
+        <div class="navigation-item"><a href="#container-3"> Craft & Play ↓ </a></div>
+        <div class="navigation-item"> <a href="#container-4"> StoryZoo ↓ </a></div>
+        <div class="navigation-item"> <a href="#container-4"> Children's Books ↓ </a></div>
+    </div>
+
+    <hr>
+
+    <div class="container">
```

Comment:
You don't have to, but I think it's a good place to introduce more specific (than div) `<section />` element, cause the container here looks like a section